vendor/osxlockdown/commands.yaml

```diff
@@ -0,0 +1,276 @@
+- title: "Verify all application software is current"
+  check_command: |
+    LASTUPDATE=$(sudo defaults read /Library/Preferences/com.apple.SoftwareUpdate | grep LastSuccessfulDate | sed -e 's@^.* "\([0-9\\-]*\) .*$@\1@'); if [ "$LASTUPDATE" = "$(date +%Y-%m-%d)" ];then exit 0; fi; exit 1
+  fix_command: |
+    sudo softwareupdate -i -a
+  enabled: false
+
+
+- title: "Enable Auto Update"
+  check_command: |
+    sudo softwareupdate --schedule | grep 'Automatic check is on'
+  fix_command: |
+    sudo softwareupdate --schedule on
+  enabled: true
+
+
+- title: "Disable Bluetooth"
+  check_command: |
+    defaults read /Library/Preferences/com.apple.Bluetooth ControllerPowerState | grep 0
+  fix_command: |
+    sudo defaults write /Library/Preferences/com.apple.Bluetooth ControllerPowerState -int 0; sudo killall -HUP blued
+  enabled: true
+
+
+- title: "Disable infrared receiver"
+  check_command: |
+    defaults read /Library/Preferences/com.apple.driver.AppleIRController DeviceEnabled | grep 0
+  fix_command: |
+    sudo defaults write /Library/Preferences/com.apple.driver.AppleIRController DeviceEnabled -int 0
+  enabled: true
+
+
+- title: "Disable AirDrop"
+  check_command: |
+    sudo defaults read com.apple.NetworkBrowser DisableAirDrop | grep 1
+  fix_command: |
+    defaults write com.apple.NetworkBrowser DisableAirDrop -bool YES
+  enabled: true
+
+
+- title: "Disable automatic setting of time and date"
```

Comment:
Because of vulnerability in the Network Time Protocol software package that has been already patched (see https://support.apple.com/en-us/HT204425)